src/components/experiments/single-view/results/ExperimentResults.test.tsx

```diff
@@ -42,15 +43,37 @@ test('renders an appropriate message for "Missing Analyses" analyses state', asy
   await expect(container.textContent).toMatch('No results are available at the moment')
 })
 
-test('renders correctly for 1 analysis datapoint', async () => {
+test('renders correctly for 1 analysis datapoint, statistically insignificant', async () => {
```

Comment:
Subtle difference: "Absence of evidence is not evidence of absence" :slightly_smiling_face: 

```suggestion
test('renders correctly for 1 analysis datapoint, not statistically significant', async () => {
```

Reply:
I love that saying